apps/jupyter/templates/services/app/ingress.yml.j2

```diff
@@ -0,0 +1,30 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: jupyter
+  namespace: "{{ namespace_name }}"
+  labels:
+    app: jupyter
+    service: app
+    version: "{{ jupyter_image_tag }}"
+  annotations:
+    cert-manager.io/issuer: "{{ acme_issuer_name }}"
+{% if jupyter_app_ip_whitelist | length %}
+    haproxy.router.openshift.io/ip_whitelist: "{{ jupyter_app_ip_whitelist | join(' ') }}"
+{% endif %}
```

Comment:
This annotation has no effect.

Reply:
You might want to look at [this annotation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#whitelist-source-range), but it will be only compatible with Kubernetes clusters using the nginx ingress controller.

Reply:
Damned yes. Will fix this thanks.

Reply:
Fixed!